account_banking_pain_base/models/account_payment_order.py

```diff
@@ -373,7 +373,7 @@ def generate_party_block(
             party_type_label = 'Creditor'
         elif party_type == 'Dbtr':
             party_type_label = 'Debtor'
-        name = 'partner_bank.partner_id.name'
+        name = 'partner_bank.account_holder'
```

Comment:
So this should end up being something link `partner_bank.account_holder or partner_bank.partner_id.name`

Reply:
Thank you very much for the review.

The idea here was to fill account_holder with the hook (for migration and new installations) and with default (for new bank accounts created when addon is installed). So by default in all situations it account_holder has the same values as partner_id.name.
Like that, when you do not care at all about account_holder, the behaviour should be exactly the same as before.
But we can add your suggestion to have a fallback in case someone puts an empty string in the field account_holder.

Reply:
Why duplicating data in all cases when 99% of them will be the same? Please keep it empty while not equal.

Reply:
This one remains to be attended.